doc/datatable/datatable.md

```diff
@@ -746,6 +747,24 @@ function (props) {
 }
 ```
 
+
+Property `disableAdvMode`is necessary to disable the advanced mode in the DataTable.
+
+Example of using `disableAdvMode`:
+```js
+...
+import { DataTable } from 'asab-webui';
+
+...
+	return (
+		<DataTable
+		...
+			disableAdvMode
```

Comment:
@Aringoaway I think it would be better to have the example like this

```
disableAdvMode={true}
```
so that is clear to everyone ,that is is a boolean (I know that it is specified in the description on line 808, but still...